src/aggregator/server/http/handlers.go

```diff
@@ -148,7 +148,7 @@ func writeResponse(w http.ResponseWriter, resp interface{}, err error) {
 
 	if err == nil {
 		w.WriteHeader(http.StatusOK)
-	} else if xerrors.IsInvalidParams(err) {
+	} else if xerrors.IsInvalidParams(err) || xerrors.IsResourceExhausted(err) {
```

Comment:
There are quite a few cases in this PR grouping invalid param with resource exhausted errors does make sense, e.g.:
- returning same status code (404 in this case);
- incrementing the same metric.

In such cases I wanted to limit the amount of changes in a single PR and I erred on maintaining externally observable behavior (status codes, published metrics) as it was up until now. This would be fixed in upcoming commits.